docs-src/go/how-to-list-workflow-executions-using-the-client-in-go.md

```diff
@@ -9,4 +9,42 @@ tags:
   - client
 ---
 
-Use [`Client.ListWorkflow`](https://pkg.go.dev/go.temporal.io/sdk/client#Client.ListWorkflow).
+There are several functions for listing [Workflow Executions](/concepts/what-is-a-workflow-execution) available on the [Temporal Go SDK](https://pkg.go.dev/go.temporal.io/sdk).
+Each function retrieves Workflow Executions according to a given [Query](/concepts/what-is-a-query) or request filters.
+
+To list all Workflow Executions, use [`ListWorkflow`](https://pkg.go.dev/go.temporal.io/sdk/client#Client.ListWorkflow) on the [Temporal Client](/concepts/what-is-a-temporal-client).
+
+```go
+resp, err := temporalClient.ListWorkflow(context.Background(), sampleListRequest)
+
+// print?
```

Comment:
do we need these print comments?

Reply:
Still trying to figure out if I need to include a print statement for the response or if it just does it of its own accord.

Reply:
I wasn't sure, so I'm gonna ask Roey or somebody